DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateGroups/GroupDelegatesViewModelTests.cs

```diff
@@ -14,27 +16,21 @@ public class GroupDelegatesViewModelTests
         private readonly DelegateGroupsSideNavViewModel expectedNavViewModel =
             new DelegateGroupsSideNavViewModel("Group name", DelegateGroupPage.Delegates);
 
-        private readonly GroupDelegate[] groupDelegates =
+        private GroupDelegate[] groupDelegates = null!;
+
+        [SetUp]
+        public void SetUp()
         {
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "a", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "b", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "c", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "d", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "e", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "f", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "g", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "h", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "i", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "j", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "k", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "l", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "m", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "n", lastName: "Surname"),
-            GroupTestHelper.GetDefaultGroupDelegate(firstName: "o", lastName: "Surname")
-        };
+            BuilderSetup.DisablePropertyNamingFor<GroupDelegate, string>(g => g.SearchableName);
+            groupDelegates = Builder<GroupDelegate>.CreateListOfSize(15)
+                .All()
+                .With(g => g.LastName = "Surname")
+                .With((g, i) => g.FirstName = NBuilderAlphabeticalPropertyNamingHelper.IndexToAlphabeticalString(i))
+                .Build().ToArray();
+        }
 
         [Test]
-        public void GroupDelegatesViewModel_should_default_to_returning_the_first_ten_delegate()
+        public void GroupDelegatesViewModel_should_default_to_returning_the_first_ten_delegates()

```

Comment:
Sorry, name needs to change again! `...returning_the_first_page_worth_of_delegates`?